trade_remedies_public/cases/v2_views/invite.py

```diff
@@ -206,6 +206,9 @@ def get_context_data(self, **kwargs):
         context["cases"] = self.client.organisations(
             self.request.user.contact["organisation"]["id"], fields=["cases"]
         )
+        context["user_cases"] = self.client.organisations(
+            self.request.user.contact["organisation"]["id"], fields=["user_cases"]
```

Comment:
Let's just use one API call and retrieve both the cases and user_cases fields, e.g.

```suggestion
            user_organisation = self.client.organisations(
            self.request.user.contact["organisation"]["id"], fields=["cases", "user_cases"]
        )
        context["cases"] = user_organisation.cases
        context["user_cases"] = user_organisation.user_cases
```

Reply:
Resolved in https://github.com/uktrade/trade-remedies-public/pull/255/commits/bff857b9e7238ee7a8b32ead19f01dd2c636402a.